src/Tasks/Microsoft.CSharp.targets

```diff
@@ -165,6 +156,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Import Project="$(MSBuildExtensionsPath)\4.0\Microsoft.NETFramework.props\ImportBefore\*" Condition="'$(ImportByWildcardBefore40MicrosoftNetFrameworkProps)' == 'true' and exists('$(MSBuildExtensionsPath)\4.0\Microsoft.NETFramework.props\ImportBefore')"/>
    </ImportGroup> 
 
+   <Import Project="$(MSBuildToolsPath)\Microsoft.ManagedLanguages.targets" />
```

Comment:
F# will need this too, eventually?

Reply:
Yeah, eventually, as this flows to master and releases. And especially once we start pilling more common code in there.